config/application.rb

```diff
@@ -154,6 +154,10 @@ class Application < Rails::Application
       }
     }
 
+    config.mediator_contact_info = {
+
```

Comment:
Is the idea that we will add this in at some point?  I don't think we symlink this anywhere so we won't be able to do it on the server. If that's the case, I wonder if we should track down that info (or make sure we have a task to know that we need to do that in order for this feature to be implemented).